packages/components/src/popover/index.js

```diff
@@ -80,24 +81,21 @@ class Popover extends Component {
 	}
 
 	componentWillUnmount() {
-		this.toggleWindowEvents( false );
-
 		clearTimeout( this.focusTimeout );
+		this.toggleAutoRefresh();
 	}
 
-	toggleWindowEvents( isListening ) {
-		const handler = isListening ? 'addEventListener' : 'removeEventListener';
-
-		window.cancelAnimationFrame( this.rafHandle );
-		window[ handler ]( 'resize', this.throttledComputePopoverPosition );
-		window[ handler ]( 'scroll', this.throttledComputePopoverPosition, true );
```

Comment:
Another possibility would be to keep those events and increase the interval timeout for advanced use-cases (like Fullscreen).

I'll give it a try in its current form though and see how it goes.

Reply:
Sounds a good alternative to have in consideration if this causes a bad performance

Reply:
I finally applied this, since I didn't like how the popovers were repositioned when scrolling the window.